ipinfo/cmd_download.go

```diff
@@ -0,0 +1,225 @@
+package main
+
+import (
+	"compress/gzip"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+const dbDownloadURL = "https://ipinfo.io/data/free/"
+
+var completionsDownload = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"-c":         predict.Nothing,
+		"--compress": predict.Nothing,
+		"-f":         predict.Nothing,
+		"--format":   predict.Nothing,
+		"-t":         predict.Nothing,
+		"--token":    predict.Nothing,
+		"-h":         predict.Nothing,
+		"--help":     predict.Nothing,
+	},
+}
+
+func printHelpDownload() {
+	fmt.Printf(
+		`Usage: %s download [<database>] [<opts>]
```

Comment:
Sorry some mistakes are here, noticed now:

- Opts to come first
- Database isnt optional
- Output should be added now, and is optional

```suggestion
		`Usage: %s download [<opts>] <database> [<output>]
```